pages/resources.js

```diff
@@ -1,5 +1,4 @@
-import { Helmet } from "@components";
-import Header from "../components/header.js";
+import { Helmet, HeaderNav } from "@components";
 import Footer from "../components/footer.js";
```

Comment:
import { Footer } from "../components/footer.jsx";

to fix compile error

Reply:
Thanks for catching that1